Signal/src/network/PushManager.m

```diff
@@ -155,7 +155,6 @@ - (void)application:(UIApplication *)application
                 success:^{
                     // TODO do we really want to mark them all as read?
                     [self markAllInThreadAsRead:notification.userInfo completionHandler:completionHandler];
-                    [[[[Environment getCurrent] signalsViewController] tableView] reloadData];
```

Comment:
This shouldn't be necessary; marking messages as read "touches" the messages' threads so the home view will get a "yap database modified" event.